test/test_subscribe.py

```diff
@@ -10,3 +10,14 @@ def test_subscribe(servers, helpers):
     assert not servers[13301].conn.eval('return _G.cond:wait(1)')[0]
     assert servers[13302].conn.eval('return membership.set_payload("foo", "bar")')[0]
     assert servers[13301].conn.eval('return _G.cond:wait(1)')[0]
+
+def test_weakness(servers, helpers):
```

Comment:
I understand your patch, but sorry this test is a Lua language test. It's a bit useless to add it. I propose to save only one-line change with a good commit message. Because a title "Collect garbage subscriptions automatically" doesn't explain you motives and actions.

Reply:
I've added a line in the changelog and updated the PR description. And I'd like to leave the test as is. It not only tests the Lua language but also checks that `membership.subscribe` doesn't own the cond.